.ci/doc/templates/print-result.tpl.java

```diff
@@ -9,7 +9,7 @@
 
   public static void main(String[] argv) {
     try {
-      kuzzle = new Kuzzle(new WebSocket("kuzzle"));
+      kuzzle = new Kuzzle(new Http("http://kuzzle:7512"));
```

Comment:
You should change the command config in `.ci/doc/config.yml` to add the protocol as environment variable
like so `cmd: SNIPPET_PROTOCOL="{{ snippet.protocol }}" /mnt/.ci/doc/test-snippet.sh {{ snippet.source }}`

And instanciate the proper protocol inside the snippet based on the environment variable `SNIPPET_PROTOCOL` instead of hardcoding the protocol to use.

You should apply this behaviour to every template